src/test/java/org/sonar/plugins/findbugs/FbContribRulesDefinitionTest.java

```diff
@@ -31,18 +33,29 @@
 
 public class FbContribRulesDefinitionTest {
 
-  @Test
-  public void test() {
+  private RulesDefinition.Repository repository;
+
+  @Before
+  public void setUp() {
```

Comment:
I have split the assertions into separate test methods.
Having multiple asserts in single test method is considered a test smell (see xUnit Test Patterns book for example)